src/Chartjs/assets/package.json

```diff
@@ -17,17 +17,17 @@
         },
         "importmap": {
             "@hotwired/stimulus": "^3.0.0",
-            "chart.js/auto": "^3.4.1"
+            "chart.js": "^3.9"
         }
     },
     "peerDependencies": {
         "@hotwired/stimulus": "^3.0.0",
-        "chart.js": "^3.4.1 <3.9"
+        "chart.js": "^3.9"
```

Comment:
For others looking at this, we can absolutely allow v4 right now. But we should do it in a minor release, not a patch release.